src/config.js

```diff
@@ -1,3 +1,13 @@
+#-------------------------------------------------------------------------------
```

Comment:
You are perfectly right... This is weird: the tool that I use to add copyright makes it wrong.
I was surprised when I saw it but did check it further.
I will fix it and submit a new pull request
Thanks